buildroot/tests/mega2560-tests

```diff
@@ -75,6 +75,22 @@ opt_enable ZONESTAR_LCD Z_PROBE_SERVO_NR Z_SERVO_ANGLES DEACTIVATE_SERVOS_AFTER_
            FILAMENT_RUNOUT_SENSOR NOZZLE_PARK_FEATURE ADVANCED_PAUSE_FEATURE Z_SAFE_HOMING
 exec_test $1 $2 "RAMPS | ZONESTAR + Chinese | MMU2 | Servo | 3-Point + Debug | G38 ..."
 
+#
+# 5 runout sensors with distinct states
```

Comment:
If you want to add an 8-extruder test you could add filament sensors to the 8-extruder test in the GTR file.

Reply:
Thanks, added